erpnext/healthcare/doctype/patient/patient.py

```diff
@@ -111,10 +111,10 @@ def make_invoice(patient, company):
 
 @frappe.whitelist()
 def get_patient_detail(patient, company=None):
-	patient_dict = frappe.db.sql(_("""select * from tabPatient where name='{0}'""").format(patient), as_dict=1)
+	patient_dict = frappe.db.sql("""select * from tabPatient where name=%s""", (patient), as_dict=1)
 	if not patient_dict:
 		frappe.throw("Patient not found")
-	vital_sign = frappe.db.sql(_("""select * from `tabVital Signs` where patient='{0}' order by signs_date desc limit 1""").format(patient), as_dict=1)
+	vital_sign = frappe.db.sql("""select * from `tabVital Signs` where patient=%s order by signs_date desc limit 1""", (patient), as_dict=1)
```

Comment:
`(patient)` should be `(patient,)`. The `values` parameter of `db.sql` takes a list (or tuple) or a dict.

Reply:
@tundebabzy `(patient)` is also ok.